x-pack/test/functional/apps/saved_objects_management/import_saved_objects_between_versions.ts

```diff
@@ -20,19 +20,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
   const retry = getService('retry');
 
-  // Failing: See https://github.com/elastic/kibana/issues/116058
-  describe.skip('Export import saved objects between versions', function () {
+  describe('Export import saved objects between versions', function () {
     before(async function () {
-      await esArchiver.load('x-pack/test/functional/es_archives/logstash_functional');
-      await esArchiver.load('x-pack/test/functional/es_archives/getting_started/shakespeare');
+      await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
       await kibanaServer.uiSettings.replace({});
       await PageObjects.settings.navigateTo();
       await PageObjects.settings.clickKibanaSavedObjects();
     });
 
     after(async () => {
-      await esArchiver.unload('x-pack/test/functional/es_archives/logstash_functional');
-      await esArchiver.unload('x-pack/test/functional/es_archives/getting_started/shakespeare');
       await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
```

Comment:
Is it a blessed way to reset Kibana to the "default" state? Maybe we should build it in the `esArchiver`?

Reply:
> Is it a blessed way to reset Kibana to the "default" state? 

AFAIK it is

> Maybe we should build it in the esArchiver?

Would make sense to me, unless I miss the subtlety doing with this very archive file. @elastic/kibana-operations wdyt?